src/server/scripts/EasternKingdoms/ZulGurub/boss_mandokir.cpp

```diff
@@ -105,14 +105,17 @@ class boss_mandokir : public CreatureScript
         {
             if (me->GetPositionZ() > 140.0f)
             {
-                killCount = 0;
                 events.ScheduleEvent(EVENT_CHECK_START, 1000);
                 if (Creature* speaker = ObjectAccessor::GetCreature(*me, instance->GetGuidData(NPC_VILEBRANCH_SPEAKER)))
                     if (!speaker->IsAlive())
                         speaker->Respawn(true);
             }
+
+            killCount = 0;
+            me->RemoveAura(SPELL_FRENZY);
```

Comment:
```suggestion
            me->RemoveAuraDueToSpell(SPELL_FRENZY);
```

Reply:
Always use RemoveAurasDueToSpell when you use spell id

Reply:
Good to know - cheers Kitzunu